testing/environments/latest.yml

```diff
@@ -3,7 +3,7 @@
 version: '2.3'
 services:
   elasticsearch:
-    image: docker.elastic.co/elasticsearch/elasticsearch:7.12.0
+    image: docker.elastic.co/elasticsearch/elasticsearch:7.14.0
```

Comment:
Should this have been updated after 7.13 and 7.14 releases?

@andresrc should we review the release process? We may also want to include there the update of the Metricbeat environments (changes like this recent one https://github.com/elastic/beats/pull/27416).